sdk/servicebus/Azure.Messaging.ServiceBus/stress/scenarios-matrix.yaml

```diff
@@ -0,0 +1,22 @@
+matrix:
+  scenarios:
+    sendrec:
+      image: Dockerfile
+      imageBuildDir: "../../../.."
+      scenarioName: "SendRec"
+    sendrecbatch:
+      image: Dockerfile
+      imageBuildDir: "../../../.."
+      scenarioName: "SendRecBatch"
+    sesssendrec:
+      image: Dockerfile
+      imageBuildDir: "../../../.."
+      scenarioName: "SessionSendRec"
+    sesssendproc:
+      image: Dockerfile
+      imageBuildDir: "../../../.."
+      scenarioName: "SessionSendProc"
+    sendproc:
```

Comment:
Not sure if it is possible, but this seems like it would be a good place to add additional parameterization for queues/topics in the future.

Reply:
Yes, these scenario matrices are new and I believe cases like that were part of the reason for adding them, I can see if I can get that to work in the next iteration.